api/src/main/java/org/eclipse/microprofile/config/Names.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2016-2020 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.eclipse.microprofile.config;
+
+/**
+ * Constants for the names of the MP-config properties that implementations must support.
+ */
+public interface Names {
```

Comment:
I would rename this to Constants. Alternatively, put the constants on the Config interface.

Reply:
Well, I was following what other projects are using to make it consistent. For instance: 
https://github.com/eclipse/microprofile-jwt-auth/blob/e0157e5a4c2a9d01b02176e08995f88eacf011f4/api/src/main/java/org/eclipse/microprofile/jwt/config/Names.java